.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,54 @@
+name: benchmarks
+
+on:
+    push:
+        branches:
+            - develop
+            - main
+    pull_request:
+        branches:
+            - develop
+            - main
+
+jobs:
+    benchmark:
+        name: benchmark
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
```

Comment:
On what version of Node are we running the benchmarks?

Reply:
good catch I have to update this. I think the current version is node 10